src/hpp/corbaserver/manipulation/constraint_graph_factory.py

```diff
@@ -32,6 +32,25 @@
 import abc
 import sys
 from .constraints import Constraints
+from .possible_grasps import PossibleGrasps
+
+
+## Class that stores grasp validation instances
+class GraspIsAllowed(object):
+    def __init__(self):
+        self.graspValidations_ = list()
+
+    ## Successively calls all the validation instances
+    #  \param grasps the set of grasp to validate
+    #  \return False if one validation fails, True otherwise
+    def __call__(self, grasps):
```

Comment:
```suggestion
class GraspIsAllowed(object):
    """Class that stores grasp validation instances"""
    
    def __init__(self):
        """
        Successively calls all the validation instances
         \param grasps the set of grasp to validate
         \return False if one validation fails, True otherwise
        """
        self.graspValidations_ = list()

    def __call__(self, grasps):
```